pkg/shared/util.go

```diff
@@ -111,3 +113,10 @@ func OwnerReferencesPresent(
 	}
 	return false
 }
+
+// StatefulSetContainers returns the array of containers are run for a given statefulSet
+func StatefulSetContainers(
+	statefulSet *appsv1.StatefulSet,
+) []v1.Container {
+	return statefulSet.Spec.Template.Spec.Containers
```

Comment:
Just to match the existing code format, let's have one blank line at the start of the function body.

Reply:
Done